test/engine-luatest/pagination_test.lua

```diff
@@ -895,8 +895,7 @@ end
 
 local no_sup = t.group('Unsupported pagination', {
             {engine = 'memtx', type = 'hash'},
-            {engine = 'memtx', type = 'bitset'},
-            {engine = 'memtx', type = 'rtree'},
```

Comment:
We still need a test to check if all iterator types except for NEIGHBOR do not support pagination.